platform.md

```diff
@@ -807,6 +813,20 @@ Where:
 - `version`, if present, SHOULD contain data uniquely identifying the particular version of the provided source
 - `metadata` MAY contain additional arbitrary data about the provided source
 
+#### `report.toml` (TOML)
+```toml
+[image]
+tags = ["<tag reference>"]
+digest = "<image digest>"
+imageID = "<imageID>"
+```
+Where:
```

Comment:
A small nit - maybe it's worth to change the order fo lines 824-828 to match the order in lines 819-821.